js/geppettoModel/model/ArrayInstance.js

```diff
@@ -8,32 +8,27 @@
  * @author Matteo Cantarelli
  */
 
-define(function (require) {
-
-  var Instance = require('./Instance');
+export default class ArrayInstance extends Instance {
 
-  function ArrayInstance (options) {
-    Instance.prototype.constructor.call(this, options);
+  constructor (options) {
+    super(options);
     this.size = options.size;
     this.length = options.size;
   }
 
-  ArrayInstance.prototype = Object.create(Instance.prototype);
-  ArrayInstance.prototype.constructor = ArrayInstance;
 
-
-  ArrayInstance.prototype.getConnections = function () {
+  getConnections = function () {
```

Comment:
@filippomc I believe something here is making the build fail
SyntaxError: /home/developer/workspace/org.geppetto.frontend/src/main/webapp/node_modules/@geppettoengine/geppetto-client/js/geppettoModel/model/ArrayInstance.js: Support for the experimental syntax 'classProperties' isn't currently enabled (20:18):
[INFO]
[INFO]   18 |
[INFO]   19 |
[INFO] > 20 |   getConnections = function () {

Reply:
See https://github.com/babel/babel/issues/8655

Reply:
yes that `= function` is wrong, missed after port to new class syntax.